--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   README.md
	modified:   source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/flat_vel/flat_vel_ddt_env_cfg.py
	modified:   source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/velocity_env_cfg.py

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/README.md b/README.md
index a318a13..443c5bd 100644
--- a/README.md
+++ b/README.md
@@ -17,10 +17,10 @@ python scripts/co_rl/ddt_train.py \
   --video \
   --video_length 300 \
   --video_interval 3000 \
-  --max_iterations 1000 \
+  --max_iterations 800 \
   --resume True \
-  --load_run 2026-05-17_14-44-23 \
-  --checkpoint model_1550.pt 
+  --load_run 2026-05-17_16-22-17 \
+  --checkpoint model_5399.pt 
 
 
 
diff --git a/source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/flat_vel/flat_vel_ddt_env_cfg.py b/source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/flat_vel/flat_vel_ddt_env_cfg.py
index cbe13ad..7459928 100644
--- a/source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/flat_vel/flat_vel_ddt_env_cfg.py
+++ b/source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/flat_vel/flat_vel_ddt_env_cfg.py
@@ -230,12 +230,12 @@ class DdtRewardsCfg:
     # --- task tracking ---
     track_lin_vel_xy_exp = RewTerm(
         func=mdp.track_lin_vel_xy_yaw_frame_exp,
-        weight=2.0,
+        weight=4.0,
         params={"command_name": "base_velocity", "std": 0.25},
     )
     track_ang_vel_z_exp = RewTerm(
         func=mdp.track_ang_vel_z_link_exp,
-        weight=1.5,
+        weight=3.5,
         params={"command_name": "base_velocity", "std": math.sqrt(0.25)},
     )
 
@@ -313,7 +313,7 @@ class DdtRewardsCfg:
     # --- height tracking ---
     base_height = RewTerm(
         func=mdp.track_pos_z_rel_exp,
-        weight=6.0,
+        weight=7.0,
         params={
             "temperature": 30.0,
             "default_height": 0.45,
@@ -322,9 +322,9 @@ class DdtRewardsCfg:
         },
     )
 
-    dof_torques_l2 = RewTerm(func=mdp.joint_torques_l2, weight=-5.0e-5)
-    dof_acc_l2 = RewTerm(func=mdp.joint_acc_l2, weight=-2.5e-7)
-    action_rate_l2 = RewTerm(func=mdp.action_rate_l2, weight= -2.5e-7)
+    dof_torques_l2 = RewTerm(func=mdp.joint_torques_l2, weight=-5.0e-3)
+    dof_acc_l2 = RewTerm(func=mdp.joint_acc_l2, weight=-12.5e-5)
+    action_rate_l2 = RewTerm(func=mdp.action_rate_l2, weight= -2.5e-51)
 
 
 # ---------------------------------------------------------------------------
@@ -439,11 +439,11 @@ class D1hDdtFlatEnvCfg(LocomotionVelocityFlatEnvCfg):
         self.events.physics_material.params["dynamic_friction_range"] = (0.6, 0.8)
 
         # commands
-        self.commands.base_velocity.ranges.lin_vel_x = (-0.05, 0.05)
-        self.commands.base_velocity.ranges.lin_vel_y = (-0.00, 0.00)
-        self.commands.base_velocity.ranges.ang_vel_z = (-0.05, 0.05)
+        self.commands.base_velocity.ranges.lin_vel_x = (-0.9, 0.9)
+        self.commands.base_velocity.ranges.lin_vel_y = (-0.02, 0.02)
+        self.commands.base_velocity.ranges.ang_vel_z = (-0.9, 0.9)
         # self.commands.base_velocity.ranges.heading = (-math.pi, math.pi)  #目标朝向
-        self.commands.base_velocity.ranges.pos_z = (-0.05, 0.05)
+        self.commands.base_velocity.ranges.pos_z = (-0.1, 0.05)
 
         # terminations
         self.terminations.terrain_out_of_bounds = None
@@ -457,7 +457,7 @@ class D1hDdtFlatEnvCfg(LocomotionVelocityFlatEnvCfg):
                  ".*_thigh",
                  ]),
                 "threshold": 100.0,
-                "start_time_s": 0.7  ,
+                "start_time_s": 0.5  ,
             },
         )
 
diff --git a/source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/velocity_env_cfg.py b/source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/velocity_env_cfg.py
index 497cc38..73ceb9b 100644
--- a/source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/velocity_env_cfg.py
+++ b/source/RL_D1h/RL_D1h/tasks/manager_based/rl_d1h/velocity_env_cfg.py
@@ -462,7 +462,7 @@ class EventCfg:
         func=mdp.reset_root_state_uniform,
         mode="reset",
         params={
-            "pose_range": {"x": (-0.0, 0.0), "y": (-0.0, 0.0), "yaw": (-0.0, 0.0)},
+            "pose_range": {"x": (-0.5, 0.5), "y": (-0.5, 0.5), "yaw": (-3.14, 3.14)},
             "velocity_range": {
                 "x": (-0.1, 0.1),
                 "y": (-0.1, 0.1),
@@ -478,8 +478,8 @@ class EventCfg:
         func=mdp.reset_joints_by_offset,
         mode="reset",
         params={
-            "position_range": (-0.0, 0.0),
-            "velocity_range": (0.0, 0.0),
+            "position_range": (-0.2, 0.2),
+            "velocity_range": (-0.2, 0.2),
         },
     )
 
@@ -498,7 +498,7 @@ class EventCfg:
         mode="interval",
         interval_range_s=(10.0, 15.0),
         params={
-            "velocity_range": {"x": (-0.01, 0.01), "y": (-0.01, 0.01), "z": (-0.01, 0.01)},
+            "velocity_range": {"x": (-0.1, 0.1), "y": (-0.0, 0.0), "z": (-0.1, 0.1)},
         },
     )
 